ssr/src/page/post_view/mod.rs

```diff
@@ -137,7 +140,7 @@ pub fn ScrollingView<NV: Fn() -> NVR + Clone + 'static, NVR>(
                     </div>
                 </Show>
 
-                <Show when=muted>
+                <Show when=show_mute_icon>
```

Comment:
If we click and `muted` is set to `false` but `show_mute_icon` will be set to false after 6 secs, which shouldn't happen right?.

Reply:
Once we mute it again it it should the mute icon again for 6 secs? In this implementation that wouldn't happen. @Natasha-GB could you confirm this behaviour